libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/MapboxOnboardRouter.kt

```diff
@@ -3,20 +3,20 @@ package com.mapbox.navigation.route.onboard
 import com.mapbox.annotation.navigation.module.MapboxNavigationModule
 import com.mapbox.annotation.navigation.module.MapboxNavigationModuleType
 import com.mapbox.geojson.Point
-import com.mapbox.navigation.base.route.Route
 import com.mapbox.navigation.base.route.Router
 import com.mapbox.navigation.navigator.MapboxNativeNavigator
 
 @MapboxNavigationModule(MapboxNavigationModuleType.OnboardRouter, skipConfiguration = true)
 class MapboxOnboardRouter(private val navigator: MapboxNativeNavigator) : Router {
 
-    override fun getRoute(origin: Point, waypoints: List<Point>, callback: (route: Route) -> Unit) {
-        TODO("not implemented")
-    }
+    override fun getRoute(
+        origin: Point,
+        waypoints: List<Point>,
+        destination: Point,
+        callback: Router.Callback
+    ) = Unit
```

Comment:
under the hood this `TODO("not implemented")` throws exception. So when it calls from tests(`core` one) it fails. We won't forget to implement it, because it's one of the main implementation